support-frontend/build.sbt

```diff
@@ -68,15 +68,19 @@ riffRaffPackageName := "frontend"
 riffRaffUploadArtifactBucket := Option("riffraff-artifact")
 riffRaffUploadManifestBucket := Option("riffraff-builds")
 riffRaffArtifactResources += (file("support-frontend/cloud-formation/cfn.yaml"), "cfn/cfn.yaml")
-
-def getFiles(f: File): Seq[(File, String)] = {
-  f match {
-    case file if file.isFile => Seq((file, file.toString.replace("support-frontend/", "")))
-    case dir if dir.isDirectory => dir.listFiles.toSeq.flatMap(getFiles)
+riffRaffArtifactResources ++= getFiles(file("support-frontend/public/compiled-assets"), "assets-static")
```

Comment:
I haven't moved the assets so they will still appear in the normal location on support.theguardian.com.  This is because the 3rd PR (todo) is to get fastly to send the requests to S3.